### ISSUE TITLE
Trying to set rotationLock causes a Type Error: Cannot set properties of undefined (setting 'mass')

### ISSUE THREAD

**Issue by davidbouchard** [quinton-ashley/p5play#313]
As of today, all my sketches that dynamically link to p5play and planck.js started showing this error whenever I try to set rotationLock = true on a Sprite: 

Uncaught TypeError: Cannot set properties of undefined (setting 'mass')

This affects the example page on the documentation as well: 

https://p5play.org/learn/sprite.html?page=9

**Comment by quinton-ashley** [comment]
yes it was, should be fixed in v3.20.2 now. Thank you for reporting the errors!
